src/workerd/io/trace.h

```diff
@@ -92,6 +92,9 @@ class TraceId final {
   // Replicates W3C Serialization
   kj::String toW3C() const;
 
+  // Return network order hex representation
+  kj::String toNEHex() const;
```

Comment:
Nit: While it's more verbose, I prefer avoiding the abbreviation here. `s/toNEHex/toNetworkOrderHex`